### ISSUE TITLE
Missing test cases

### ISSUE THREAD

**Issue by jajik** [modcluster/mod_proxy_cluster#93]
Currently, tests are missing:

* multiple balancers
* issue 63 tests

**Comment by rhusar** [comment]
@jajik Is this regarding https://github.com/modcluster/mod_proxy_cluster/tree/main/test ? Do we have instructions how to run these?

**Comment by jajik** [comment]
Yes. And more or less yes. There is a README within `test/native/` that describes it a bit. It might be worth it to mention it in the main README.

**Comment by jajik** [comment]
Several testcases were added in https://github.com/modcluster/mod_proxy_cluster/pull/283
